<launch>
    <include file="$(find gps)/launch/run.launch"/>
    <!-- <include file="$(find unitree_sdk2)/launch/run.launch"/> -->

    <include file="$(find mqtt_commx)/launch/run.launch"/>
    
    <!-- <node pkg="tf2_ros" type="static_transform_publisher" name="velodyne_rslidar" args="0 0 0 0 0 0 velodyne rslidar"/> -->
    <!-- <node pkg="data_comm" name="can_comm" type="can_comm"/> -->

    <!-- <node pkg="global_plan_sim" type="local_path_plan" name="local_path_plan" output="screen"/> -->
    <!-- 路径规划 -->
    <include file="$(find pathplan)/launch/run.launch"/>
    <node pkg="pathtrack" type="carctr_decision" name="carctr_decision" output="screen"/>
    <node pkg="pathtrack" type="pathtrack" name="pathtrack" output="screen">
        <!-- <param name="run_enable" value="false"/> -->
        <param name="angle_offset" value="5"/>
        <param name="sync" value="false"/>
    </node>
    <!-- <node pkg="pathtrack" type="selfturnctr" name="selfturnctr" output="screen"/> -->
    <!--multi_car-->
    <include file="$(find multi_car)/launch/run.launch"/>
    <!-- <node pkg="rviz" type="rviz" name="rviz_global_plan_sim" args="-d $(find global_plan_sim)/test.rviz" output="screen" respawn="true">
        <param name= "init_path" value="my"/>
        <param name = "arrive_flag" value = "false"/>
    </node> -->
    
    <include file="$(find car_ctr)/launch/run.launch"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find multi_car)/show.rviz" output="screen"/>
    <!-- <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"/> -->
</launch> 
